backend/src/main/java/gov/cdc/usds/simplereport/db/model/Facility.java

```diff
@@ -78,35 +80,60 @@ public String getFacilityName() {
 		return facilityName;
 	}
 
+    public DeviceSpecimen getDefaultDeviceSpecimen() {
+        return defaultDeviceSpecimen;
+    }
+
 	public DeviceType getDefaultDeviceType() {
-		return defaultDeviceType;
+        return defaultDeviceSpecimen == null ? null : defaultDeviceSpecimen.getDeviceType();
 	}
 
 	public void setDefaultDeviceType(DeviceType defaultDeviceType) {
-		this.defaultDeviceType = defaultDeviceType;
+        throw new IllegalArgumentException();
 	}
 
 	public List<DeviceType> getDeviceTypes() {
 		// this might be better done on the DB side, but that seems like a recipe for weird behaviors
-		return configuredDevices.stream()
+        return configuredDeviceSpecimens.stream()
 			.filter(e -> !e.isDeleted())
+                .map(DeviceSpecimen::getDeviceType)
+                .filter(e -> !e.isDeleted())
 			.collect(Collectors.toList());
 	}
 
-	public void addDeviceType(DeviceType newDevice) {
-		configuredDevices.add(newDevice);
-	}
-
-	public void addDefaultDeviceType(DeviceType newDevice) {
-		this.defaultDeviceType = newDevice;
-		addDeviceType(newDevice);
-	}
-
-	public void removeDeviceType(DeviceType existingDevice) {
-		configuredDevices.remove(existingDevice);
-		if (defaultDeviceType != null && existingDevice.getInternalId().equals(defaultDeviceType.getInternalId())) {
-			defaultDeviceType = null;
-		}
+    public List<DeviceSpecimen> getDeviceSpecimens() {
+        return configuredDeviceSpecimens.stream()
+                .filter(e -> !(e.isDeleted() || e.getSpecimenType().isDeleted() || e.getDeviceType().isDeleted()))
+                .collect(Collectors.toList());
+    }
+
+    public void addDeviceSpecimen(DeviceSpecimen ds) {
+        configuredDeviceSpecimens.add(ds);
+    }
+
+    public void addDefaultDeviceSpecimen(DeviceSpecimen newDefault) {
+        addDeviceSpecimen(newDefault);
+        defaultDeviceSpecimen = newDefault;
+    }
+
+    public void removeDeviceSpecimen(DeviceSpecimen existing) {
+        configuredDeviceSpecimens.remove(existing); // count on this being in one hibernate session
+    }
+
+    public void removeDeviceType(DeviceType existingDevice) {
+        Iterator<DeviceSpecimen> i = configuredDeviceSpecimens.iterator();
+        UUID removedId = existingDevice.getInternalId();
+        while (i != null && i.hasNext()) {
+            DeviceType d = i.next().getDeviceType();
+            if (d.getInternalId().equals(removedId)) {
+                i.remove();
+                if (defaultDeviceSpecimen != null
```

Comment:
True, I think. I'm trying to figure if there was a reason I put it in there—I think it was a somewhat misplaced instinct to optimize. I may not change it, since this method is deprecated with prejudice, but it would improve readability, so I probably should.